src/models/Wallet.ts

```diff
@@ -48,4 +57,18 @@ export default class Wallet implements ISigner {
       throw err;
     }
   }
+
+  /**
+   * We just need to pass a valid sendTo address belonging to that network for the
+   * qtumjs-wallet library to calculate the maxQtumSend amount.  It does not matter what
+   * the specific address is, as that does not affect the value of the
+   * maxQtumSend amount
+   */
+  private maxQtumSendToAddress = (networkName: string) => {
```

Comment:
nit: can be a `@computed get func()`

Reply:
when I try to do so I get a message stating that a get accessor cannot have parameters. The only way I see around that is to have this.main.network passed in as part of the constructor on a wallet model object, but I don't think it makes sense. Do you have another idea in mind?

Reply:
ya sorry forgot about no params in computed values. looks like the networkName needs to be passed here since it doesn't have access to the `NetworkStore`.